### ISSUE TITLE
training with larger batchsize

### ISSUE THREAD

**Issue by zwyzwy** [tianweiy/CenterPoint#218]
hello ，
if i training with 8 nvidia 3090 GPUs and setting samples_per_gpu=8 , shall i change the learning rate to lr_max=0.006 ?

**Comment by tianweiy** [comment]
you can. I think the model is not too sensitive to the learning rate
